docs/testing.md

```diff
@@ -20,82 +20,104 @@ To execute the bats framework:
-
-#### Examples
-
-    ansible-playbook pipelines/install_pipeline.yml -e forklift_state=up -e pipeline_os=debian9 -e pipeline_type=foreman -e pipeline_version=nightly
-    ansible-playbook pipelines/upgrade_pipeline.yml -e forklift_state=up -e pipeline_os=centos7 -e pipeline_type=katello -e pipeline_version=3.10
+optional arguments:
+  -h, --help  show this help message and exit
+```
 
-### Creating Pipelines
+Individual pipelines also have help texts:
+```console
+$ ./bin/forklift install --help
+usage: forklift install [-h] [-v] [-e EXTRA_VARS] [--state FORKLIFT_STATE]
+                        [--os PIPELINE_OS] [--type PIPELINE_TYPE]
+                        [--version PIPELINE_VERSION]
+
+Run an install pipeline
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -v, --verbose         verbose output
+  --state FORKLIFT_STATE
+                        Forklift state to ensure
+  --os PIPELINE_OS      Operating system to install, like centos7, debian10 or
+                        ubuntu1804
+  --type PIPELINE_TYPE  Type of pipeline, like foreman, katello or luna
+  --version PIPELINE_VERSION
+                        Version to install, like nightly, 1.24 or 3.14
+
+advanced arguments:
+  -e EXTRA_VARS, --extra-vars EXTRA_VARS
+                        set additional variables as key=value or YAML/JSON, if
+                        filename prepend with @
+```
 
-If you wish to add a new version of an existing pipeline (e.g. a new Katello release), you only have to add the corresponding vars files to `pipelines/vars/`.
+Pipelines typically have a state which defaults to `up`. Other valid values are `rebuild` and `destroy`. The latter one is useful to clean up which pipelines don't do by themselves.
 
-For Katello 3.11, you'd be adding the following two files:
+For example to run a Foreman Nightly installation pipeline on Debian Buster:
 
-`pipelines/vars/katello_3.11.yml`:
-```yaml
-forklift_name: pipeline-katello-3.11
-forklift_boxes:
-  pipeline-katello-3.11-centos7:
-    box: centos7
-    memory: 8192
-  pipeline-proxy-3.11-centos7:
-    box: centos7
-    memory: 3072
-katello_repositories_version: '3.11'
-katello_repositories_pulp_version: '2.19'
-foreman_repositories_version: '1.21'
-foreman_client_repositories_version: "{{ foreman_repositories_version }}"
+```console
+$ ./bin/forklift install --os debian10 --type foreman --version nightly
+... lots of output
+$ ./bin/forklift install --os debian10 --type foreman --version nightly --state destroy
 ```
 
-`pipelines/vars/katello_upgrade_3.11.yml`:
-```yaml
-katello_version_start: '3.9'
-katello_version_intermediate: '3.10'
-katello_version_final: '{{ katello_version }}'
+Similarly a Katello Nightly upgrade pipeline on CentOS 7:
+
+```console
+$ ./bin/forklift upgrade --os centos7 --type katello --version nightly
+... lots of output
+$ ./bin/forklift upgrade --os centos7 --type katello --version nightly --state destroy
```

Comment:
A little more explanation of why the second run is used would make it more understandable for someone starting to use it.